catalog/video/video-mpegts-transformer/src/main/java/org/codice/alliance/transformer/video/MpegTsInputTransformer.java

```diff
@@ -72,6 +81,8 @@
 
     private Integer subsampleCount = DEFAULT_SUBSAMPLE_COUNT;
 
+    private Function<MpegStreamType, String> streamTypeToString = new StreamTypeToString();
```

Comment:
btw, it simplifies down to:

private Function<MpegStreamType, String> streamTypeToString = MpegStreamType::toString;